drivers/RFM69/new/RFM69_new.cpp

```diff
@@ -297,6 +355,8 @@ LOCAL void RFM69_interruptHandling(void)
 						RFM69.currentPacket.payloadLen = readingLength;
 						RFM69.ackReceived = RFM69_getACKReceived(RFM69.currentPacket.header.controlFlags);
 						RFM69.dataReceived = !RFM69.ackReceived;
+
+						RFM69_DEBUG(PSTR("RFM69:MH:l=%" PRIi8 " pl=%" PRIi8 "\n"), readingLength, (RFM69.ackReceived ? 1 : 0));
```

Comment:
You're not support to serial print from interrupts

Reply:
ok